dashboards-observability/public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -892,6 +892,34 @@ export const Explorer = ({
           [GROUPBY]: [{ bucketSize: '', bucketOffset: '' }],
           [AGGREGATIONS]: [],
         };
+      case VIS_CHART_TYPES.LogsView: {
+        const dimensions = [
+          ...statsToken.aggregations
```

Comment:
I think you do not need to duplicate the aggregation array by using spread here. You are using a map function over an existing array, that should return a new array anyway - hence it is not going to affect the existing array - which is actually the reason behind creating this new array using the spread operator syntax.

Reply:
Resolved.